tensorflow/examples/android/build.gradle

```diff
@@ -69,4 +69,8 @@ task copyNativeLibs(type: Copy) {
 
 copyNativeLibs.dependsOn buildNative
 assemble.dependsOn copyNativeLibs
-assembleDebug.dependsOn copyNativeLibs
+tasks.whenTaskAdded { task ->
```

Comment:
Can you add a comment to explain the reasoning behind this change please? I vaguely remember having to previously work around a bug in Gradle here -- just want to make sure this change will work under all circumstances going forward.